build/scripts/deploy-msbuild.ps1

```diff
@@ -0,0 +1,51 @@
+
+# This script will ensure that the MSBuild toolset of the specified 
+# version is downloaded and deployed to Binaries\Toolset\MSBuild
+#
+# This is a no-op if it is already present at the appropriate version
+
+param ()
+set-strictmode -version 2.0
+$ErrorActionPreference="Stop"
+
+$version = "0.1.7-alpha"
+$repoDir = [IO.Path]::GetFullPath((join-path $PSScriptroot "..\.."))
+$toolsDir = join-path $repoDir "Binaries\Toolset"
+$scratchDir = join-path $toolsDir "Scratch"
+
+try {
+
+    mkdir $toolsDir -ErrorAction silent | out-null
+    mkdir $scratchDir -ErrorAction silent | out-null
+    pushd $toolsDir
+
+    $scratchFile = join-path $scratchDir "msbuild.txt"
+    $scratchVersion = gc -raw $scratchFile -ErrorAction SilentlyContinue
+    if ($scratchVersion -eq ($version + [Environment]::NewLine)) { 
+        exit 0
+    }
+
+    $tempFileName = "msbuild-$version.zip"
+    $tempFile = join-path $scratchDir $tempFileName
+    if (-not (test-path $tempFile)) { 
+        $url = "https://jdashstorage.blob.core.windows.net/msbuild/msbuild-$version.zip" 
```

Comment:
That's deliberate at this point.  I didn't want to add anything until we needed it.  Otherwise it's pretty much impossible to determine if I added it correctly.